Running-Mastodon/PgBouncer-guide.md

```diff
@@ -13,13 +13,18 @@ Note that you can check "PgHero" in the administration view to see how many Post
 Installing PgBouncer
 -----
 
-On Ubuntu:
+On Ubuntu/Debian:
 
     sudo apt install pgbouncer
 
-Restarting:
+Then:
 
-    sudo service pgbouncer restart
+    nano /etc/default/pgbouncer
+    START=1
```

Comment:
I believe that even in Ubuntu you need the START=1 if you want PgBouncer to start on boot. At least, according to [this](http://blog.tomnod.com/install-pgbouncer-ubuntu).
So, I am thinking of not changing the PR.